tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -119,11 +127,11 @@ private static bool IsDatadogAssembly(string path, out Assembly cachedAssembly)
 
         private static void SetDatadogAssembly(string path, Assembly cachedAssembly)
         {
-            for (var i = 0; i < _assemblies.Length; i++)
+            for (var i = 0; i < _assemblies!.Length; i++)
```

Comment:
Why even bother with the nullable analysis 😛

Reply:
Hah, I was adding comments until I saw this. I will stop now.